go.mod

```diff
@@ -94,7 +94,7 @@ require (
 	github.com/holiman/uint256 v1.2.4 // indirect
 	github.com/inconshreveable/mousetrap v1.1.0 // indirect
 	github.com/ipfs/bbloom v0.0.4 // indirect
-	github.com/ipfs/boxo v0.19.0 // indirect
```

Comment:
This was a wrong dependency bump that caused Go to be required to be 1.21. See https://github.com/onflow/flow-go/pull/5822 and https://discord.com/channels/613813861610684416/1235013191633993749/1235015098930565130